CSS/style.css

```diff
@@ -377,3 +385,68 @@ p {
   padding: 23px 40px 23px 0;
   cursor: pointer;
 }
+
+.contact::before {
+  content: "";
+  position: absolute;
+  background-color: #6070ff;
+  background-image: url("../Image/contact-bg.svg");
+  background-size: contain;
+  background-position: right;
+  background-repeat: no-repeat;
+  border-top-left-radius: 5rem;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.contact {
+  position: relative;
+  padding: 6.4375rem 1.5rem 1.5rem;
+  z-index: 10;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact__title {
+  color: #fff;
+  text-align: center;
+}
+
+.contact__dsc {
+  color: #ebebff;
+  text-align: center;
+  margin: 0.8rem 0 2.8rem;
+}
+
+.contact__container form li input {
+  width: 100%;
+  border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin: 10px 0;
+  height: 48px;
+  padding: 0 1rem;
+}
+
+.contact__container form li textarea {
+  width: 100%;
+  border-radius: 8px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  margin-top: 8px;
+  padding: 1rem;
+  resize: none;
+}
+
+.contact__container form li .btn {
+  width: 162px;
+  height: 48px;
+  padding: 12px;
+  color: #6070ff;
+  cursor: pointer;
+  margin: 18px 0;
+}
```

Comment:
- [ ] Please stick to the design as much as possible. Kindly add a border radius to the button to match the design template
Design template
![Screenshot 2022-08-23 194844](https://user-images.githubusercontent.com/94551484/186239895-e80510b7-298b-4dac-a714-f114352978f0.png)

Your design
![Screenshot 2022-08-23 194825](https://user-images.githubusercontent.com/94551484/186239904-62156330-5623-449f-8e9e-2ef47ba44d8c.png)
.

- [ ] Please consider using flexbox to layout the form elements.